### PR TITLE
Bundle signed grub+shim in grub2-efi package

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by Itxaka** [comment]
probably superseedes https://github.com/kairos-io/packages/pull/135 and makes system/shim deprecated

**Review by mauromorales — APPROVED** [review]
yup I think it's better both together to see the branching caused by the arch :+1:

**Comment by Itxaka** [comment]
This also provides the shim for both arches, which is not the case in the shim package, it only provides it for x86.
Also provides both the signed grub and shim which are required for secureboot.

also matches with https://github.com/kairos-io/elemental-cli/pull/44 which uses those paths to use the grub artifacts during the install

File: packages/system/grub2-efi/build.yaml [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+steps:
+- zypper in grub2-arm64-efi shim-susesigned
+includes:
+- usr/share/grub2/arm64-efi/
+- usr/share/efi/.*/*.efi
+- usr/share/efi/.*/*.der
+  {{else}}
 steps:
-- mkdir -p /data
-- wget  https://download.opensuse.org/repositories/isv:/Rancher:/Elemental/15.4/noarch/grub2-{{.Values.arch}}-efi-{{.Values.pversion}}.noarch.rpm -O /package.rpm
```
Comment (Itxaka) [inline packages/system/grub2-efi/build.yaml:8]: this doesnt seem to exist anymore? Plus now elemental uses upstream grub IIRC.

**Review by mudler — APPROVED** [review]

**Pull request merged**